pkg/controller/command/command_upgrade.go

```diff
@@ -76,6 +95,19 @@ func getContainerImage(containers []core.Container, name string) string {
 	return ""
 }
 
+func (r *ReconcileCommand) checkIfImageChangedAgain(command *contrail.Command) (bool, error) {
```

Comment:
Maybe information about to which version we are upgrading should be in the status of Command.